tests/test_smart_ptr.py

```diff
@@ -1,125 +0,0 @@
-from pybind11_tests import ConstructorStats
```

Comment:
(deleted this testcase temporarily, will reintroduce once shared_ptr MI stuff also works)